### ISSUE TITLE
ITC-3 sends AggregatedStatus too early, when using core 3.1.2

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_validator#290]
When using core 3.1.2 the tlc sends an AggregatedStatus before the intial supervisor Wathdog has been acknowledged.

The watchdog must be ack'ed before other types are send.

**Comment by emiltin** [comment]
@otterdahl can you provide input on this?

**Comment by emiltin** [comment]
Similiar to #291, the issue here is that the device sends the Aggregated status before acknowledging the Watchdog it receives during the handshake. Do we want to enforce that messages are acked, before movingto the next phase of the handshake?

**Comment by otterdahl** [comment]
With version 3.1.2 it isn't clear that messages needs to be acked before moving on to the next phase.

With 3.1.3 we clarified that messages can be sent asynchronously, *"i.e. while the site or supervision system is waiting for an answer to a previously sent message it can can continue to send messages"* (this is mainly due to performance reasons - something we discovered early on).

With 3.1.4 we clarified that the initial version message is the expection to this. We must have the ability check the version message and block the connection is the requirements aren't met.

In conclusion. I don't think we can require this.

**Comment by emiltin** [comment]
ok

**Comment by SwarcoPalm** [comment]
I'm just reviewing open issues and I believe this can be closed?

**Comment by emiltin** [comment]
Closing, for core >= 3.1.4 we verify the sequence, but not for earlier versions.
